src/Authenticator/CookieAuthenticator.php

```diff
@@ -224,7 +225,8 @@ protected function _createCookie($value): CookieInterface
             $data['path'],
             $data['domain'],
             $data['secure'],
-            $data['httpOnly']
+            $data['httponly'],
+            $data['samesite']
```

Comment:
Using [Cookie::create()](https://api.cakephp.org/4.1/class-Cake.Http.Cookie.Cookie.html#create) instead of `new Cookie()` would avoid having to pass each option separately.